app/src/main/java/jp/kztproject/rewardedtodo/di/todo/TodoDatabaseModule.kt

```diff
@@ -1,6 +1,7 @@
 package jp.kztproject.rewardedtodo.di.todo
 
 import android.app.Application
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/2c63e5b5afa3521e4c927845d93d2e0ccb187422/app/src/main/java/jp/kztproject/rewardedtodo/di/todo/TodoDatabaseModule.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>